public/docs/_examples/dependency-injection/dart/lib/app_component_1.dart

```diff
@@ -1,17 +1,17 @@
 // Early versions
 
 // #docregion
-import 'package:angular2/core.dart';
+import 'package:angular2/angular2.dart';
```

Comment:
I have no idea where exactly I saw this. It was a while back when the public API was reworked about 2 months ago. I don't think it's too important.